sdk/attestation/azure-security-attestation/tests/test_policy_getset.py

```diff
@@ -17,7 +17,7 @@
 # ----------------------
 
 from logging import fatal
-from typing import ByteString
+from typing import ByteString, Any
```

Comment:
This only seems to be used when type checking though, so you may want to do a TYPE_CHECKING import here and `if TYPE_CHECKING:` block after imports
```suggestion
from typing import Any
```